test/image/mocks/heatmap_shared_categories.json

```diff
@@ -0,0 +1,49 @@
+{
+  "data": [{
+    "type": "heatmap",
+    "x": ["Team A", "Team B", "Team C"],
+    "xaxis": "x",
+    "y": ["Game Three", "Game Two", "Game One"],
+    "z": [
+      [0.1, 0.3, 0.5],
+      [1, 0.8, 0.6],
+      [0.6, 0.4, 0.2]
+    ],
+    "yaxis": "y"
+  }, {
+    "type": "heatmap",
+    "x": ["Team B", "Team C"],
+    "xaxis": "x",
+    "y": ["Game Three", "Game Two", "Game One"],
+    "z": [
+      [0.3, 0.5],
+      [0.8, 0.6],
+      [0.4, 0.2]
+    ],
+    "yaxis": "y2"
+  }],
+  "layout": {
```

Comment:
Nice mock! Can you try using `coloraxis` from https://github.com/plotly/plotly.js/pull/3803 so that we don't have two overlapping (ugly) colorbars? Thanks!

Reply:
Done in 793c2a346e07d4dfa5504a20dd95d6fa47290ff0